dmscripts/helpers/auth_helpers.py

```diff
@@ -0,0 +1,16 @@
+import os
+import subprocess
+import yaml
+
+
+def get_dev_auth_token(api, stage):
+    DM_CREDENTIALS_REPO = os.environ.get('DM_CREDENTIALS_REPO')
+    creds = subprocess.check_output([
+        "{}/sops-wrapper".format(DM_CREDENTIALS_REPO),
+        "-d",
+        "{}/vars/{}.yaml".format(DM_CREDENTIALS_REPO, stage)
+    ])
+    auth_tokens = yaml.load(creds)[api]['auth_tokens']
+    dev_token = [token for token in auth_tokens if token.startswith('D')]
```

Comment:
If this is running on Jenkins it should use the Jenkins token beginning with 'j', surely?